python/serialbox/serialization.py

```diff
@@ -292,6 +292,9 @@ def __repr__(self):
     def __str__(self):
         return self.name + '[ ' + ' '.join(['{0}:{1}'.format(*i) for i in self.metainfo.items()]) + ' ]'
 
+    def __eq__(self, other):
+        return self.name == other.name and self.metainfo == other.metainfo
```

Comment:
No, that is a different use case.
I'm replicating here the same interface we have in C++: two savepoint object (whose goal is to define points in the program execution, not to hold data) are checked against each other to see if they represent the same point in time (e.g.) in two different program executions.
For instance, you want to see if two savepoints both represent the point just before horizontal advection of step 4. Whether two program executions saved the same data at this savepoint or not is a different story.
We already use == between savepoints in the verification framework of C++ with this meaning and the Python API should replicate this meaning.